src/ontology/mondo-edit.obo

```diff
@@ -453558,7 +453235,6 @@ created_by: https://orcid.org/0000-0001-5208-3432
 id: MONDO:0100282
 name: SC phocomelia syndrome
 comment: Reason: duplicate. This will be merged with MONDO:0100253 Roberts-SC phocomelia syndrome
```

Comment:
could you please remove these comments too?
thanks!

Reply:
done